vespa/deployment.py

```diff
@@ -110,30 +115,49 @@ def _run_vespa_engine_container(
                 self.container = client.containers.get(application_name)
                 self.container.restart()
             except docker.errors.NotFound:
-                logging.debug("Start a Docker container: "
-                              "image: {image}, "
-                              "mem_limit: {mem_limit}, "
-                              "name: {name}, "
-                              "hostname: {hostname}, "
-                              "volumes: {volumes}, "
-                              "ports: {ports}".format(
-                    image=self.container_image,
-                    mem_limit=container_memory,
-                    name=application_name,
-                    hostname=application_name,
-                    volumes={disk_folder: {"bind": "/app", "mode": "rw"}},
-                    ports={8080: self.local_port}
-                ))
-                self.container = client.containers.run(
-                    self.container_image,
-                    detach=True,
-                    mem_limit=container_memory,
-                    name=application_name,
-                    hostname=application_name,
-                    privileged=True,
-                    volumes={disk_folder: {"bind": "/app", "mode": "rw"}},
-                    ports={8080: self.local_port},
-                )
+                if disk_folder:
```

Comment:
My understanding is that this option will be removed when deploying through zip is validated.

Reply:
correct